megamek/src/megamek/MegaMek.java

```diff
@@ -255,41 +242,43 @@ public static String getMegaMekSHA256() {
     public static String getMemoryUsed() {
         long heap = Runtime.getRuntime().totalMemory();
         long free = Runtime.getRuntime().freeMemory();
-        long used = (heap - free) / 1024;
-        return commafy.format(used) + " kB"; //$NON-NLS-1$
+        return commafy.format((heap - free) / 1024) + " kB";
     }
 
     /**
      * This function redirects the standard error and output streams to the
      * given File name.
      *
-     * @param logFileName
-     *            The file name to redirect to.
+     * @param logFileName The file name to redirect to.
      */
     private static void redirectOutput(String logFileName) {
-        try {
-            System.out.println("Redirecting output to " + logFileName); //$NON-NLS-1$
-            String sLogDir = PreferenceManager.getClientPreferences()
-                    .getLogDirectory();
-            File logDir = new File(sLogDir);
-            if (!logDir.exists()) {
-                logDir.mkdir();
+        getLogger().info(MegaMek.class, "redirectOutput",
+                "Redirecting output to " + logFileName);
+        String sLogDir = PreferenceManager.getClientPreferences().getLogDirectory();
+        File logDir = new File(sLogDir);
+        if (!logDir.exists()) {
+            if (!logDir.mkdir()) {
+                getLogger().error(MegaMek.class, "redirectOutput",
+                        "Error in creating directory ./docs. We know this is annoying, and apologise. "
```

Comment:
Is this really creating a directory called `./docs`? Wouldn't it be `./logs`?

Reply:
Copy-paste error, fixing